lib/emergency_banner/display.rb

```diff
@@ -42,12 +43,13 @@ def link_text
 
     def content
       return @content if defined? @content
+
       @content = begin
-        client.hgetall("emergency_banner").try(:symbolize_keys)
-      rescue StandardError => e
-        GovukError.notify(e)
-        nil
-      end
+                   client.hgetall("emergency_banner").try(:symbolize_keys)
+                 rescue StandardError => e
+                   GovukError.notify(e)
+                   nil
+                 end
```

Comment:
I've been avoiding this indentation style, as it looks quite awkward to me. What do you make of it?

Reply:
Ok